packages/openapi-to-graphql/src/oas_3_tools.ts

```diff
@@ -280,43 +280,50 @@ function buildUrl(server: ServerObject): string {
 }
 
 /**
- * Returns object | array where all object keys are sanitized. Keys passed in
- * exceptions are not sanitized.
+ * Returns object/array/scalar where all object keys (if applicable) are
+ * sanitized.
  */
-export function sanitizeObjKeys(
-  obj: object | Array<any>,
-  exceptions: string[] = []
-): object | Array<any> {
-  const cleanKeys = (obj: object | Array<any>): object | Array<any> => {
+export function sanitizeObjectKeys(
+  obj: any, // obj does not necessarily need to be an object
+  caseStyle: CaseStyle = CaseStyle.camelCase
+): any {
+  const cleanKeys = (obj: any): any => {
+    // Case: no (response) data
     if (obj === null || typeof obj === 'undefined') {
       return null
+
+      // Case: array
     } else if (Array.isArray(obj)) {
       return obj.map(cleanKeys)
+
+      // Case: object
     } else if (typeof obj === 'object') {
       const res: object = {}
-      for (let key in obj) {
-        if (!exceptions.includes(key)) {
-          const saneKey = sanitize(key, CaseStyle.camelCase)
-          if (Object.prototype.hasOwnProperty.call(obj, key)) {
-            res[saneKey] = cleanKeys(obj[key])
-          }
-        } else {
-          res[key] = cleanKeys(obj[key])
+
+      for (const key in obj) {
+        const saneKey = sanitize(key, caseStyle)
+
+        if (Object.prototype.hasOwnProperty.call(obj, key)) {
+          res[saneKey] = cleanKeys(obj[key])
         }
       }
+
       return res
+
+      // Case: scalar
     } else {
       return obj
     }
   }
+
   return cleanKeys(obj)
 }
 
 /**
  * Desanitizes keys in given object by replacing them with the keys stored in
  * the given mapping.
  */
-export function desanitizeObjKeys(
+export function desanitizeObjectKeys(
```

Comment:
We actually have another function called `desanitizeObjectKeys()` in the same file...

I realized that the other `desanitizeObjectKeys()` function was not useful so I ended up removing it but I was surprised to find it.